inspirehep/modules/workflows/workflows/article.py

```diff
@@ -317,26 +262,180 @@
 ]
 
 
-ADD_MARKS = [
+STOP_IF_EXISTING_SUBMISSION = [
     IF(
-        article_exists,
+        is_submission,
+        IF(
+            is_marked('is-update'),
+            NOTIFY_ALREADY_EXISTING
+        )
+    )
+]
```

Comment:
This can be removed.

Reply:
I removed `CHECK_IF_MERGE_AND_STOP_IF_SO` that is doing something different compared to what it says